pkg/util/github/commit.go

```diff
@@ -28,3 +24,17 @@ func (c *Client) GetLastCommit() (*github.RepositoryCommit, error) {
 
 	return commits[0], nil
 }
+
+func (c *Client) GetCommitTree(ref *github.Reference, commitInfo *repo.CommitInfo) (*github.Tree, error) {
```

Comment:
The function name and the action of the code do not match. `Get` <-> `CreateTree`. 

`BuildCommitTree` or `CreateCommitTree`?